data_safe_haven/resources/workspace/ansible/desired_state.yaml

```
@@ -148,6 +148,53 @@
         creates: /etc/audit/rules.d/50-privileged.rules
       notify: Restart auditd
 
+    - name: Copy ClamAV configuration
+      ansible.builtin.copy:
+        src: etc/clamav/clamd.conf
+        dest: /etc/clamav/clamd.conf
+        mode: '0644'
+
+    # This also generates a default freshclam.conf if one does not exist
+    - name: Enable and start freshclam service
+      ansible.builtin.systemd:
+        name: clamav-freshclam
+        state: started
+        enabled: true
+
+    - name: Set freshclam private mirror
+      ansible.builtin.lineinfile:
+        path: /etc/clamav/freshclam.conf
+        line: "{{lookup('file', '/etc/clamav/freshclam-mirror.conf')}}"
```

```suggestion
        line: "{{ lookup('file', '/etc/clamav/freshclam-mirror.conf') }}"
```

Jinja2 likes spaces inside braces. I think it can fail sometimes if you don't.

This will read the contents of `freshclam-mirror.conf` and write it to `freshclam.conf`?

---

Yes, we want to append `PrivateMirror clamav.$SRE_FQDN` to this file, but SRE_FQDN isn't available to Ansible as a variable. Could be refactored as part of #2075.